### ISSUE TITLE
Don't warn about single arg use when there's a second arg in a reader conditional

### ISSUE THREAD

**Issue by mk** [clj-kondo/clj-kondo#1914]
The following code in a cljc file

```clojure
(let [scope *ns*]
  (or (#{:default} scope)
      #?(:clj (keyword? scope))))
```

Gives `warning: Single arg use of or always returns the arg itself `

Rewriting this is tedious and the macro expand also will generate the optimal code so it might be better to not warn at all.
